src/store/index.tsx

```diff
@@ -0,0 +1,11 @@
+export interface PlayerState {
```

Comment:
nit: this is just personal preference, but id put these in the App.tsx file since it is that file that is consuming these interfaces

Reply:
let's talk about this later